index.html

```diff
@@ -38,6 +39,131 @@ <h1 class="primary-text">I’m Richmond Glad to see you!</h1>
             <li class="angelistic"><a href="#"><img src="./logos/angelist.png" alt="github-logo" /></a>
             </li>
         </ul>
+      </section>
+      <section class="works-container">
+        <div class="projects-cards">
+          <div class="card">
```

Comment:
- [ ] In the work section, the card background is not matched with the Figma design. The card background color will be white which will be different from your page background. Please try to match your page's work section with the figma design as much as possible.

Your design             |  Figma Design
:-------------------------:|:-------------------------:
![c1](https://user-images.githubusercontent.com/70483835/164936253-80ee1537-0f3e-4633-8134-06a594065cbb.png) | ![c2](https://user-images.githubusercontent.com/70483835/164936274-5954af61-f57e-483d-85c2-0cb1bdfaae58.png)